services/workflows-service/src/alert/alert.controller.external.ts

```diff
@@ -107,11 +107,15 @@ export class AlertControllerExternal {
         alertDetails: alertDefinition.description,
         subject: counterparty.business
           ? {
+              type: 'business',
               id: counterparty.business.id,
               name: counterparty.business.companyName,
+              correlationId: counterparty.business.correlationId,
             }
           : {
+              type: 'counterparty',
```

Comment:
A counterparty can be a business, individual, or something else, but the types are just `business` and `counterparty`?

Reply:
That's what I understood from Nitzan.